src/Protocol/Protocol_1_8.cpp

```diff
@@ -1827,6 +1827,49 @@ int cProtocol_1_8_0::GetParticleID(const AString & a_ParticleName)
 
 
 
+UInt32 cProtocol_1_8_0::GetProtocolMobID(int a_MobType)
+{
+	switch (a_MobType)
+	{
+		case mtBat:                   return 65;
+		case mtBlaze:                 return 61;
+		case mtCaveSpider:            return 59;
+		case mtChicken:               return 93;
+		case mtCow:                   return 92;
+		case mtCreeper:               return 50;
+		case mtEnderDragon:           return 63;
+		case mtEnderman:              return 58;
+		case mtGhast:                 return 56;
+		case mtGiant:                 return 53;
+		case mtGuardian:              return 68;
+		case mtHorse:                 return 100;
+		case mtIronGolem:             return 99;
+		case mtMagmaCube:             return 62;
+		case mtMooshroom:             return 96;
+		case mtOcelot:                return 98;
+		case mtPig:                   return 90;
+		case mtRabbit:                return 101;
+		case mtSheep:                 return 91;
+		case mtSilverfish:            return 60;
+		case mtSkeleton:              return 51;
+		case mtSlime:                 return 55;
+		case mtSnowGolem:             return 97;
+		case mtSpider:                return 52;
+		case mtSquid:                 return 94;
+		case mtVillager:              return 120;
+		case mtWitch:                 return 66;
+		case mtWither:                return 64;
+		case mtWolf:                  return 95;
+		case mtZombie:                return 54;
+		case mtZombiePigman:          return 57;
+		default:                      return a_MobType;
+	}
```

Comment:
Default is not a good idea. We want `-Wenum` errors to tell us if any cases are missing.
```suggestion
	}
	UNREACHABLE("Unsupported mob type");
```